lib/plugins/inventory.js

```diff
@@ -88,11 +94,7 @@ function inject (bot, { version, hideErrors }) {
       const item = window.findItemRange(start, end, window.selectedItem.type, window.selectedItem.metadata, true, window.selectedItem.nbt)
 
       if (item && item.stackSize !== item.count) { // something to join with
-        if (noWaiting) {
-          await clickWindow(item.slot, 0, 0)
-        } else {
-          await Promise.all([clickWindow(item.slot, 0, 0), once(window, `updateSlot:${item.slot}`)])
-        }
+        await clickWindow(item.slot, 0, 0)
```

Comment:
`noWaiting` never did what it advertised it did, and now that we don't use it at all we should remove it